src/cli/conio.cpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * conio.cpp
+ * 
+ * Particle serial input and output for command line use
+ * 
+*/
```

Comment:
Please make sure all asterisks in this are aligned (last line is off).

Please also use the Doxygen formatting (include `@file`)

Reply:
Done